src/System.Data.SqlClient/tests/FunctionalTests/SqlParameterTest.cs

```diff
@@ -38,5 +38,29 @@ public void ParameterPrecisionOnBaseType()
             Assert.Equal(10, parameter.Precision);
             Assert.Equal(5, parameter.Scale);
         }
+
+        [Fact]
+        public void CreateParameterWithXmlSchema()
```

Comment:
This test should cover more scenarios for these 3 APIs. e.g. Null values and calling set on already initiated _xmlSchemaCollection. Please add more scenarios that cover different paths to ensure no scenarios fail.

Reply:
Added a couple more tests covering setting to empty and non-empty values to make sure rountrips work.